src/command_modules/azure-cli-interactive/azure/cli/command_modules/interactive/azclishell/_dump_commands.py

```diff
@@ -66,67 +62,27 @@ def load_arguments(self):
                 loader._update_command_definitions()  # pylint: disable=protected-access
 
 
-class LoadFreshTable(object):
+# pylint: disable=too-few-public-methods
+class FreshTable(object):
     """
     this class generates and dumps the fresh command table into a file
     as well as installs all the modules
     """
-    def __init__(self):
-        self.command_table = None
-        self.shell_ctx = None
+    command_table = None
```

Comment:
Why is this a class variable rather than an instance variable like it was before?

Reply:
We were using a global variable to keep an instance of the table as a singleton.
Keeping the table in a class as a variable accomplishes the same and seems to be preferable in python.